access/base/delete.md

```diff
@@ -44,7 +44,9 @@ Delete a specified **permission**
 
 #### ❌ Raises
 * `SecurityRequestError`<br>
-  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is equal to `4`.
```

Comment:
I think we can get rid of the `returned by IRRSMO00 part` to reduce noise. I think the following gets the message across:

Also, for clarity, I think we should be referring to the `SAF Return Code`.

```markdown
Raises `SecurityRequestError` when the **SAF Return Code** of a **Security Result** is equal to `4`.
```

Reply:
This issue has been resolved.